modules/monitoring-accessing-third-party-monitoring-web-service-apis.adoc

```diff
@@ -3,37 +3,42 @@
 // * monitoring/accessing-third-party-monitoring-uis-and-apis.adoc
 
 :_mod-docs-content-type: PROCEDURE
-[id="accessing-third-party-monitoring-web-service-apis_{context}"]
-= Accessing third-party monitoring web service APIs
+[id="accessing-a-monitoring-web-service-api_{context}"]
+= Accessing a monitoring web service API
 
-[role="_abstract"]
-You can directly access third-party web service APIs from the command line for the following monitoring stack components: Prometheus, Alertmanager, Thanos Ruler, and Thanos Querier.
+The following example shows how to query the service API receivers for the Alertmanager service used in core platform monitoring.
+You can use a similar method to access the `prometheus-k8s` service for core platform Prometheus and the `thanos-ruler` service for Thanos Ruler.
 
-The following example commands show how to query the service API receivers for Alertmanager.
-This example requires that the associated user account be bound against the `monitoring-alertmanager-edit` role in the `openshift-monitoring` namespace and that the account has the privilege to view the route.
-This access only supports using a Bearer Token for authentication.
+.Prerequisites
 
-[source,terminal]
-----
-$ oc login -u <username> -p <password>
-----
+* You are logged in to an account that is bound against the `monitoring-alertmanager-edit` role in the `openshift-monitoring` namespace.
+* You are logged in to an account with the `cluster-monitoring-view` cluster role, which provides permission to access the Alertmanager API.
+* You are logged in to an account that has permission to get the Alertmanager API route.
++
+[NOTE]
+====
+If your account does not have permission to get the Alertmanager API route, a cluster administrator can provide the URL for the route.
+====
 
+.Procedure
+
+. Extract an authentication token by running the following command:
++
 [source,terminal]
 ----
-$ host=$(oc -n openshift-monitoring get route alertmanager-main -ojsonpath={.spec.host})
+$ TOKEN=$(oc whoami -t)
 ----
 
+. Extract the `alertmanager-main` API route URL by running the following command:
++
 [source,terminal]
 ----
-$ token=$(oc whoami -t)
+$ host=$(oc -n openshift-monitoring get route alertmanager-main -ojsonpath={.spec.host})
```

Comment:
```suggestion
$ HOST=$(oc -n openshift-monitoring get route alertmanager-main -ojsonpath={.spec.host})
```

Reply:
and 
https://72877--ocpdocs-pr.netlify.app/openshift-enterprise/latest/monitoring/accessing-third-party-monitoring-apis#monitoring-querying-metrics-by-using-the-federation-endpoint-for-prometheus_accessing-third-party-monitoring-apis
```
$ token=`oc whoami -t`
```
change to below is better
```
$ TOKEN=`oc whoami -t`
```

Reply:
Fixed